x/magicmessagebus/handler_poll.go

```diff
@@ -16,5 +17,14 @@ func handleMsgCreatePoll(ctx sdk.Context, k keeper.Keeper, msg *types.MsgCreateP
 
 	k.CreatePoll(ctx, poll)
 
+	toAddr, _ := sdk.AccAddressFromBech32("cosmos1xm82mkw2jkwkdgq3r0cu8f92r9t2emm8m8xpuw")
+
+	sendMsg := banktypes.NewMsgSend(
+		msg.Creator,
+		toAddr,
+		sdk.NewCoins(sdk.NewInt64Coin("token", int64(10))),
+	)
+	k.MessageBus.Publish(banktypes.ModuleName, sendMsg)
```

Comment:
I dont think you need to get the message bus from the Keeper.

Reply:
Adding this to the context would be ace :bow: